.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -31,7 +31,8 @@ changes warrant it!
 ## Further Comments
 <!-- Optional, any edits/updates should preferably be written here. -->
 
-## Acknowledgement
-- [ ] I have read the [Contributing Guidelines](https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP))
+## Acknowledgements
+- [ ] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
+- [ ] I have added an entry describing the changes from this PR to the [Changelog](https://docs.manim.community/en/latest/changelog.html)
```

Comment:
Perhaps this should not only link the web URL of the changelog, but also reference .rst file that the docs are built from. 

`[ ] I have added an entry describing the changes from this PR to the [Changelog](https://docs.manim.community/en/latest/changelog.html)  at docs/source/changelog.rst

Reply:
Good idea! db8e11bc